### PR TITLE
Implement component Logo

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Implement component Logo 
image

https://6684faab2d0c99a3e93bf145-mfdlwiitqj.chromatic.com/?path=/story/components-logo--default

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,32 +1,21 @@
-import { RouterProvider } from 'react-router-dom';
-import router from './routers';
-import { useEffect } from 'react';
-import ProductAPIService from './services/ProductAPIService';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import Home from './pages/Home/Home';
+import Cart from './pages/Cart/Cart';
+import Details from './pages/Details/Details';
+import Order from './pages/Order/Order';
```
Comment (hungdo-agilityio) [inline src/App.tsx]: Please renaming file into `index.tsx` so that we can `import Home from './pages/Home`

File: src/components/Logo/Logo.tsx [hunk 2]
```diff
@@ -1,5 +1,26 @@
+// Logo.tsx
```
Comment (hungdo-agilityio) [inline src/components/Logo/Logo.tsx]: useless comment

File: src/components/Logo/Logo.tsx [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+import { Link } from 'react-router-dom';
+import companyLogo from '../../assets/images/logo.svg';
+import styles from './logo.module.css';
+
 const Logo = () => {
-  return <></>;
+  return (
```
Comment (hungdo-agilityio) [inline src/components/Logo/Logo.tsx]: shorthand

File: src/App.tsx [hunk 4]
```diff
@@ -1,32 +1,21 @@
...
-  }, []);
-  // example end
-
   return (
-    <>
-      <RouterProvider router={router} />
-    </>
+    <BrowserRouter>
+      <Routes>
+        <Route>
```
Comment (hungdo-agilityio) [inline src/App.tsx]: Is this `<Route>` redundant

File: src/components/Logo/Logo.stories.tsx [hunk 5]
```diff
@@ -1,9 +1,17 @@
 import type { Meta, StoryObj } from '@storybook/react';
 import Logo from './Logo';
+import { BrowserRouter } from 'react-router-dom';
 
 export default {
   component: Logo,
   title: 'Components/Logo',
+  decorators: [
+    Story => (
+      <BrowserRouter>
```
Comment (hungdo-agilityio) [inline src/components/Logo/Logo.stories.tsx]: Please add decorator globally instead of just for this component

File: src/App.tsx (hunk 1 above)
Comment (thanhtrantruong47) [reply src/App.tsx]: Fixed

File: src/components/Logo/Logo.tsx (hunk 2 above)
Comment (thanhtrantruong47) [reply src/components/Logo/Logo.tsx]: Fixed

File: src/App.tsx (hunk 4 above)
Comment (thanhtrantruong47) [reply src/App.tsx]: Fixed

File: src/components/Logo/Logo.stories.tsx (hunk 5 above)
Comment (thanhtrantruong47) [reply src/components/Logo/Logo.stories.tsx]: Fixed

**Review by thanhnguyen-agilityio — APPROVED** [review]

**Pull request merged**